### DISCUSSION TITLE
Simple api to get device performance

### DISCUSSION THREAD

**Discussion by zsilverzweig** [creativecreatorormaybenot/performance#22] [Ideas]
There are a lot of variables to assess app performance, which I don't understand or want to understand. As a junior dev, I'd love to be able to rely on `performance` and do something like: performance.level==high ? animatedWidget : basicWidget.

**Comment by creativecreatorormaybenot** [comment]
Hi, this is out of scope for this package (the scope of this package is displaying absolute performance numbers).

For this kind of feedback, you should file a feature request over at https://github.com/flutter/flutter 👍🏼
